js/5etools-main.js

```diff
@@ -866,6 +866,10 @@ const betteR205etoolsMain = function () {
 		}
 
 		const $body = $("body");
+
+		// Text for force external request warning
+		const ferWarning = `<div><b style="color:red">WARNING: You are currently fetching data from an external source, rather than the script's inbuilt data. External data may not be compatible with your current script version. Do not use this feature unless necessary.</b></div>`
```

Comment:
remove UI changes, as we don't want these for debug features
(if we do want them, they should be dynamically enabled/disabled by toggling the appropriate value, not static at build time)